lib/hyrax/publisher.rb

```diff
@@ -155,6 +155,10 @@ class Publisher
     # @macro a_registered_event
     register_event('object.acl.updated')
 
+    # @since 3.4.0
+    # @macro a_registered_event
+    register_event('object.acl.deleted')
```

Comment:
seeing `object.acl` in this context is making me wonder if this is the right event namespace. we're going to publish this for `AdminSet` resources, which are not Objects. i was just encountering a similar issue that motivated to #5429, so i'm curious.

i know this is borrowing from the `object.acl.updated` event above, but i'm wondering whether both shouldn't change...

or i guess this part might be moot in light of https://github.com/samvera/hyrax/pull/5428#issuecomment-1028547693

Reply:
I was thinking the same thing, so I think `object.acl.updated` should get explanded.  But outside the scope of this PR.  I'll remove this event since I closed #5428.